smac/utils/io/traj_logging.py

```diff
@@ -87,22 +89,25 @@ def add_entry(self, train_perf: float, incumbent_id: int,
             id of incumbent
         incumbent: Configuration()
             current incumbent configuration
+        budget: float
+            budget used in intensifier to limit TA (default: 0)
         """
         ta_runs = self.stats.ta_runs
         ta_time_used = self.stats.ta_time_used
         wallclock_time = self.stats.get_used_wallclock_time()
         self.trajectory.append(TrajEntry(train_perf, incumbent_id, incumbent,
-                               ta_runs, ta_time_used, wallclock_time))
+                               ta_runs, ta_time_used, wallclock_time, budget))
         if self.output_dir is not None:
-            self._add_in_old_format(train_perf, incumbent_id, incumbent,
+            self._add_in_old_format(train_perf, incumbent_id, incumbent, budget,
```

Comment:
Sorry, I wasn't precise enough. The budget should be added only to the trajectory class and the alljson format. The other two are standardized formats which do not support adding additional keys/columns.